README.MD

```diff
@@ -65,6 +65,14 @@ mongo_db_bundle:
         ssl: false
         connectTimeoutMS: 3000 # default null (no timeout)
         readPreference: primaryPreferred # see https://docs.mongodb.com/manual/reference/read-preference/#primary for info
+      
+      ssl_client_name: # see https://docs.mongodb.com/manual/tutorial/configure-ssl-clients/
+        uri: 'mongodb://host1:3062,host2'
+        password: ''
+        authSource: ''
+        ssl: true
+        driver_options:
```

Comment:
Actually I don't agree with this approach.
I'd find a better solution referring to a generic factory for driver options.
The service should implement a specific interface.